Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:4.0
+// swift-tools-version:5.0
```

Comment:
It looks like this change breaks compatibility with Swift 4.2, so it would require a new major release and updating the documented minimum requirements in the README.

Is there any way to fix these problems without going to 5.0 of the tools?  Maybe version 4.2?

Reply:
Unfortunately it was first introduced in Swift 5: https://github.com/apple/swift-evolution/blob/master/proposals/0236-package-manager-platform-deployment-settings.md

Reply:
Is there any chance for a new major release in the near future? 

It would help to use https://github.com/watson-developer-cloud/swift-sdk 
with Swift-Package-Manager, what is really convenient in use. 

THX in advance